locale/fi/sidenavigation.json

```diff
@@ -0,0 +1,74 @@
+{
+  "title": "Sivunavigaatio (SideNavigation)",
```

Comment:
For the English names, the custom in ds-site has been to write them as "common language" instead of the pascalcase format. So this would be "side navigation"

Reply:
Check!